README.md

```diff
@@ -1,30 +1,85 @@
 # NUpbr
+
 This repository holds code to generate a configurable, Physically Based Rendered (PBR) football field.
 
 ![Field Example](./docs/outputs/goals_example.gif)
 
 # Requirements
-+ [Blender](https://www.blender.org/download/) (2.79)
+
+-   [Blender](https://www.blender.org/download/) (2.79)
 
 # Usage
-## Generating field UV map
-A UV map for the field can be generated by running [`generate_uv.py`](./field_uv/generate_uv.py). This will use the configuration listed in [`scene_config.py`](./pbr/config/scene_config.py).
 
-## Building the scene
-To generate the scene with default field UV map, the `pbr` script should be run through the Blender Python API. That is, `blender --python pbr.py` while in the pbr directory. This will create a scene, rendering a ball, goals and a field depending on the HDR metadata.
+## Set Up
+
+-   Clone this repo
+-   Change into the cloned `NUpbr` directory, then into `pbr`
+-   Install dependencies by running `ensure_dependencies.py` _using the Python binary installed with Blender_. You may have to run it twice: first to install Pip, and then to install the dependencies.
+-   Download the latest [resources.zip](https://github.com/NUbots/NUpbr/releases) file from Releases and copy the `resources` directory from it into the `NUpbr` root directory. More resources are available on the NUbots NAS.
```

Comment:
Will make a release with the file after this is merged.

Reply:
We'll still have to be careful of sharing copy written material, but sounds good.

Reply:
We'll still have to be careful of copywritten material (and probably provide references to where we got the material where possible), but looks good!

Reply:
Yeah. Do we have a ball model with an open licence that we can use?

Reply:
Yeah the balls that are just the colour maps should be fine- they were just UV maps taken from Google. The HDRs were free-to-use from HDR Haven.

Reply:
Might still be better to air on the side of caution and not make a release for the resources at all

Reply:
True. Would we be able to link to a zip small on the NAS with everything you need to run, where the link is only accessible to us internally?

Reply:
That should be possible to achieve